app/controllers/project.py

```diff
@@ -204,17 +204,51 @@ def get(self):
         page = request.args.get('page', 1, type=int)
         per_page = request.args.get('per_page', 10, type=int)
         keywords = request.args.get('keywords', '')
+        disabled = request.args.get('disabled')
+        project_type = request.args.get('project_type')
+        organisation = request.args.get('organisation')
```

Comment:
Hey @MutegekiHenry, let us change `disabled` to`is_active`: Which is type boolean, brings active when set to True, disabled ones when set to False and does not filter when not sent hence its default can be None.

Reply:
Hi @Mubangizi disabled is already a boolean field and challenge with changing it atm is its being used by other endpoints that set it to true or false by either the user or the admin. but essentially what you have described is what is happening